serverlessworkflow/modules/ROOT/pages/_common-content/sonataflow-metrics.adoc

```diff
@@ -0,0 +1,134 @@
+== Overview
```

Comment:
@jianrongzhang89 this  common sonataflow_metrics document is great, many thanks.
My only observation is that the order of occurrence of each metric in the document, is not the same as the one being shown in the initial paragraph, which somehow corresponds with the workflow "natural" life-cycle.

see:
![Screenshot from 2024-12-12 13-18-27](https://github.com/user-attachments/assets/b08768d0-9311-4d87-abf0-204acbb12026)

Reply:
Good suggestion. Done!